### PR TITLE
set-reach-router

### PR DESCRIPTION
**Pull request by wildcod**
#77

### CODE REVIEW

File: view-src/src/AppRoutes.js [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
 
 class AppRoutes extends Component {
     render() {
         return (
-            <Switch>
-                <Route path="/" exact component={Home} />
-                <Route component={NotFound} />
-            </Switch>
+            <Router>
+                <Home path="/" exact  />
```
Comment (abhishek71994) [inline view-src/src/AppRoutes.js]: Put this in a router.js file and import it here.

File: view-src/src/routes.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import React from 'react'
+import { Router } from "@reach/router"
+
```
Comment (abhishek71994) [inline view-src/src/routes.js]: Send the route design from here.

**Comment by wildcod** [comment]
I'm not be able to correct this conflict can you please help?

**Comment by abhishek71994** [comment]
Sure. Let's discuss this over on discord :apple:

**Comment by abhishek71994** [comment]
I have added some fix, closing this PR.

**Pull request closed without merging**